### PR TITLE
feat: add package handler and lint codes

### LINKED ISSUE #13: Report test coverage and make sure it is high enough
Something like https://app.codecov.io/gh/apache/apisix-go-plugin-runner

### LINKED ISSUE #14: The CI on master branch is broken

### PR DESCRIPTION
**Pull request by shuaijinchao**
FIX #14 
FIX #13

### CODE REVIEW

File: .github/workflows/unit-test-ci.yml [hunk 1]
```diff
@@ -43,6 +43,8 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Set up dependencies
-        run: make setup
+        run: make setup install
+      - name: Lint codes
+        run: make lint
```
Comment (spacewander) [inline .github/workflows/unit-test-ci.yml:48]: Better to move lint to a separate workflow

File: Makefile [hunk 2]
```diff
@@ -17,11 +17,32 @@
...
-	python3 -m pip install click --ignore-installed
-	python3 -m pip install pytest --ignore-installed
+	python3 -m pip install -r requirements.txt --ignore-installed
+
 
 .PHONY: test
 test:
-	python3 -m pytest -v tests
+	pytest --version || python3 -m pip install pytest-cov
+	python3 -m pytest --cov=apisix/runner tests
```
Comment (spacewander) [inline Makefile:26]: I just look at the coverage.
84% is a great score to share. Shall we upload it to codecov.io and put a badger on the README? We can do it in the next PR.

File: .github/workflows/unit-test-ci.yml (hunk 1 above)
Comment (shuaijinchao) [reply .github/workflows/unit-test-ci.yml]: done

File: Makefile (hunk 2 above)
Comment (shuaijinchao) [reply Makefile:26]: OK, good idea

File: .github/workflows/runner-lint.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  RunPython:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
```
Comment (spacewander) [inline .github/workflows/runner-lint.yml:34]: We don't need to lint in many Python versions. It will slow down the CI.

Comment (shuaijinchao) [reply .github/workflows/runner-lint.yml]: fixed

**Comment by shuaijinchao** [comment]
@spacewander please review

**Review by spacewander — APPROVED** [review]

**Pull request merged**